src/utils/notifications.ts

```diff
@@ -142,7 +149,13 @@ export function getMarkedUnreadState(room: Room): boolean | undefined {
     return currentStateStable ?? currentStateUnstable;
 }
 
-export async function setUnreadMarker(room: Room, client: MatrixClient, unread: boolean): Promise<void> {
+/**
+ * Sets the marked_unread state of the given room
+ * @param room The room to set
+ * @param client MatrixClient object to use
+ * @param unread The new marked_unread state of the room
+ */
```

Comment:
Could you expand this documentation to explain a bit more about what it means to set the "marked unread state"? In particular: it sets some account-data, and the value of that account data then changes the result of `determineUnreadState` [somehow}?

Reply:
Done in f25c19145b.